docs/getting_started_linux_cmake.md

```diff
@@ -0,0 +1,99 @@
+# Getting Started on Linux with CMake
+
+## Prerequisites
+
+### Install CMake
+
+Install CMake version >= 3.13:
+
+```shell
```

Comment:
Should this just be `sh`?

Reply:
`sh` is one of the aliases for `shell`: https://github.com/github/linguist/blob/40992ba7f86889f80dfed3ba95e11e1082200bad/lib/linguist/languages.yml#L4984-L5004 , so I think they are equivalent.

I've just been using `shell` consistently, but I don't have a strong opinion on which of the aliased names to use.